assets/src/blocks/CarouselHeader/useSlides.js

```diff
@@ -8,8 +8,12 @@ import { useState, useEffect } from '@wordpress/element';
  * - Adds an *exit* transition class for the `active` element
  * - Adds a listener for `ontransitionend` to the `active` element
  *
- * @param {Array} slidesRef
+ * @param {Array}  slidesRef
+ * @param {*}      lastSlide
+ * @param {*}      containerRef
  * @param {Object} options
+ *
+ * @return {*} not exactly sure
```

Comment:
Is this an official return type? 😁 Can't we just put `object`?

Reply:
So I was looking at the function and I could not figure out what the return type was, the `*` just means any

Reply:
Ah okay 🙂 the description should still be improved I think, maybe you can just write "functions for the carousel header slides"